README.md

```diff
@@ -125,11 +125,21 @@ It is expected that you have downloaded the pretrained weights as described abov
 The finetuning requires at least one GPU with ~24 GB memory (GTX 3090). Follow the instructions in the script to efficiently fit your GPU memory.
 Note: For some GPU models you might need to set `torch.backends.cuda.enable_flash_sdp(False)` (see comments at the top of the script).
 
-More details about each finetuning method and how you can apply it to your own data can be found in our how-to guides:
+More details about each finetuning method and how you can apply it to your own data can be found in our technical how-to guides.
+
+### Finetuning How-To Guides
+
+These technical tutorials illustrate how to run the finetuning code.
 
 - [Finetune with LoRA](howto/finetune_lora.md)
 - [Finetune with Adapters](howto/finetune_adapter.md)
 
+### Understanding Finetuning -- Conceptual Tutorials
+
+Looking for conceptual tutorials and explanations? We have some additional articles below:
+
+- [Understanding Parameter-Efficient Finetuning of Large Language Models: From Prefix Tuning to LLaMA-Adapters](https://lightning.ai/pages/community/article/understanding-llama-adapters/)
```

Comment:
I think it would be good to include this reference inside `howto/finetune_adapter.md` as well (somewhere near the introductory text perhaps?). I thought I did that already but it must have gotten lost somehow. 

And of course we would love to include the LoRA article as well when it comes out.

Reply:
Good call! Added that. (And I made a note to add the LoRA article as well later this week after that has gone live!)